dist_tplink_v3/install-linux.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/env bash
+
+if ! command -v adb &> /dev/null; then
```

Comment:
We should make this match our new unified install script for orbic which just has install.sh and not separate scripts for mac and linux.